home/models.py

```diff
@@ -26,3 +27,31 @@ class Article(Page):
     content_panels = Page.content_panels + [
         StreamFieldPanel('body')
     ]
+
+@register_snippet
+class Footer(models.Model):
+    title = models.CharField(max_length=255, blank=True)
+    logos = StreamField([
+        ('image', ImageChooserBlock())
+    ], null=True)
```

Comment:
The `logos`, `navigation` and `essential` fields are currently all mandatory required fields (despite your `null=True`)...

The confusion is discussed [here](https://github.com/wagtail/wagtail/issues/4306).

In addition to `null=True`, `blank=True` and `required=True`... both `StreamField` and `StreamBlock` have `min_num` and `max_num` arguments (discussed [here](https://github.com/wagtail/wagtail/issues/5175)).

Reply:
quite confusing. i've attempted to make only `title` and `essential` required fields, the rest are optional.